src/components/action-pad/action-pad.tsx

```diff
@@ -155,6 +139,8 @@ export class ActionPad
   // --------------------------------------------------------------------------
 
   connectedCallback(): void {
+    connectLocalized(this);
```

Comment:
were these missing?

Reply:
this is a duplicate. will get this removed.